js/test/test.js

```diff
@@ -245,6 +286,7 @@ async function testExchange (exchange) {
     for (let i = 0; i < codes.length; i++) {
         if (codes[i] in exchange.currencies) {
             code = codes[i];
+            break;
```

Comment:
there was no point the iteration to continue, we need the first occurrence, as it's sorted by priority.